LBHFSSPublicAPI/V1/Helpers/UrlHelper.cs

```diff
@@ -0,0 +1,23 @@
+using System.Text.RegularExpressions;
+using System.Web;
+
+namespace LBHFSSPublicAPI.V1.Helpers
+{
+    public static class UrlHelper
+    {
+        public static string DecodeParams(string term)
+        {
+            while (true)
+            {
+                var regex = new Regex(@"%20|%25");
+                Match m = regex.Match(term);
```

Comment:
ASP.NET Core already decodes the urls. If the consumer has them double encoded it's not out issue, but theirs.

We provide the interface to the API that follows world-wide standards and we should expect the API to be used that way. If someone uses the interface incorrectly _(like double encoding the url),_ then it's on them to fix it so they could use the API.

Using a car manufactoring analogy:
`"Fixing" the API by adding a "double_encoded_url" support into a "single_encoded_url" API  interface in case user passes in incorrect url format.`

is the same as 

`"Fixing" the car by adding a diesel support into a petrol car in case a user fills up the car with incorrect fuel type.`

If you want to use a Petrol car, fill it up with Petrol not diesel. I think the same should go with APIs.